### PR TITLE
[IOS-2738]Follow MoPub code style guidelines

### PR DESCRIPTION
**Pull request by YueVungle**
Need to update our MoPub adapter code to follow MoPub code style guidelines:
https://developers.mopub.com/networks/integrate/mopub-network-mediation-guidelines/#code-style-guidelines

IOS-2738

### CODE REVIEW

**Review by shiv-vungle — APPROVED** [review]
LGTM

File: Vungle/VungleRouter.m [hunk 1]
```diff
@@ -303,19 +321,24 @@ - (void)requestBannerAdWithPlacementID:(NSString *)placementID size:(CGSize)size
     }
 }
 
-- (BOOL)isAdAvailableForPlacementId:(NSString *) placementId {
+- (BOOL)isAdAvailableForPlacementId:(NSString *) placementId
```
Comment (crbinsf) [inline Vungle/VungleRouter.m]: Extra space between `)` and `placementID`

**Review by crbinsf — COMMENTED** [review]
Changes look good - one minor format issue to address. Also, just want to confirm that property declaration changes have been made to remove portions of declaration that should be considered as ‘default’ (i.e., ‘assign’ for BOOL, ‘strong’ for some objects).

File: Vungle/VungleRouter.m (hunk 1 above)
Comment (YueVungle) [reply Vungle/VungleRouter.m]: @crbinsf Thanks a lot for pointing out this error, fixed. :)

**Comment by YueVungle** [comment]
> Changes look good - one minor format issue to address. Also, just want to confirm that property declaration changes have been made to remove portions of declaration that should be considered as ‘default’ (i.e., ‘assign’ for BOOL, ‘strong’ for some objects).

@crbinsf Hi Clarke, thanks a lot for pointing out the format issue. And for property declaration changes have been made to remove portions of declaration that should be considered as ‘default’ (i.e., ‘assign’ for BOOL, ‘strong’ for some objects), this is required by MoPub code style guidance ^_^  (https://developers.mopub.com/networks/integrate/mopub-network-mediation-guidelines/#methods-1):

_Properties: Do not specify strong or assign attributes for properties.They are default attributes.
@property (nonatomic) BOOL isValid;   // => assign by default
@property (nonatomic) NSString *name; // => strong by default_

**Review by crbinsf — APPROVED** [review]
Looks good. Thanks!

**Review by ashinagawa — APPROVED** [review]
LGTM

**Pull request merged**